lib/csv_process.rb

```diff
@@ -0,0 +1,106 @@
+# frozen_string_literal: true
```

Comment:
The main source of annoyance from this file is the fact that each record in the csv comes in as a hash with both attributes for teacher and school, so we just have 1 big hash with the following keys: 

:first_name, :last_name, :education_level, :email, :more_info, :personal_website, :snap, :status, :school_id, :school_name, :school_city,  :school_state, :school_website, :school_grade_level, :school_type, :school_tags, :school_nces_id

This means we need to manually specify which hash values we want when we create a new teacher/school and that makes the code quite ugly. Feedback specific to this issue would be greatly appreciated. 

Also, `teacher_db = Teacher.where(email: row[:email]).or(Teacher.where(snap: row[:snap]))` didn't work with the current code, but we will incorporate it when once we do a major refactor of this file.